entrypoint.sh

```diff
@@ -24,15 +24,29 @@ if [[ ! "$1" =~ ^(fmt|init|plan|validate)$ ]]; then
   exit 1
 fi
 
+if [ "$2" ]; then
+  input_raw="$2"
+fi
+
 comment="$4"
+file="$5"
```

Comment:
does this work if input is empty?
Then $2 should be empty and everything is shifted left, so file would be $4

Reply:
The parameters are passed unconditionally here:

https://github.com/Jimdo/terraform-pr-commenter/blob/0e578e8f76efd65b4156cebea85b0469be074a7f/action.yml#L26-L34

I would expect github action to not do any magic behind the scenes. So there is no shifting being done it will just be an empty string passed.